qa/rpc-tests/test_framework/blocktools.py

```diff
@@ -58,20 +64,26 @@ def serialize_script_num(value):
         r.append(0x80 if neg else 0)
     elif neg:
         r[-1] |= 0x80
+    r.insert(0, len(r))
```

Comment:
it also would not include the length-prefix

Reply:
Again, I don't think is required. If someone is pushing the number onto the stack, they will append this value.